tsconfig.json

```diff
@@ -2,6 +2,7 @@
   "compilerOptions": {
     "module": "ES2015",
     "moduleResolution": "node",
+    "esModuleInterop": true,
```

Comment:
Note that this PR currently flips `esModuleInterop` to `true`. I'm not sure if this is supported in google3.

Without `esModuleInterop`, node wasm tests fail. This is because the Node tests no longer run against the node bundle, and instead they're run against the JavaScript output files. These javascript files are incorrect when `esModuleInterop` is not set because the js file for the WASM bindings [exports a single function as the default export](https://github.com/tensorflow/tfjs/blob/master/tfjs-backend-wasm/wasm-out/tfjs-backend-wasm.d.ts#L40-L42), which behaves differently in node vs in the bundled web files. I'm still trying to understand why.

Reply:
This PR no longer flips `esModuleInterop` to true.